docs/Developer-Guide.md

```diff
@@ -212,10 +212,14 @@ $ sudo systemctl restart systemd-journald
 >
 > - You should only do this step if you are testing with the latest version of the agent.
 
-The rust-agent is built with a static linked `musl.` To configure this:
+The rust-agent is built with a static linked `musl.` The default rust component used is `musl`, but on `ppc64le` and `s390x`,
+`gnu` should be used. To configure this:
 
 ```
-rustup target add x86_64-unknown-linux-musl
+export ARCH=$(uname -m)
+[ ${ARCH} == "ppc64le" ] || [ ${ARCH} == "s390x" ] && export LIBC=gnu || export LIBC=musl
+[ ${ARCH} == "ppc64le" ] && export ARCH=powerpc64le
```

Comment:
Thanks! for consistency i believe "$ " should be pre-appended before each command
also, is setting ARCH=powerpc64le may mess some of the build scripts?

Reply:
Yes, I appreciate there wasn't one there before (which is a bug), but please could you add the shell prompt prefix as documented in:

- https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks

Reply:
Thanks @snir911 and @jodh-intel. And if ARCH=powerpc64le, we didn't need to ln musl-g++

Reply:
> also, is setting ARCH=powerpc64le may mess some of the build scripts?

@snir911, AFAIU, the build scripts are taking care of handling the `ARCH`.

Reply:
> Yes, I appreciate there wasn't one there before (which is a bug), but please could you add the shell prompt prefix as documented in:
> 
> * https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#code-blocks

I wonder why we have this requirement. It does not add much clarity for anybody capable of correctly parsing

```
[ ${ARCH} == "ppc64le" ] || [ ${ARCH} == "s390x" ] && export LIBC=gnu || export LIBC=musl
```

On the other hand, it makes it more difficult to copy-paste code blocks from the documentation into a shell.